app/src/commands/apply.rs

```diff
@@ -206,6 +206,8 @@ impl ComtryaCommand for Apply {
 
                     let action = action.inner_ref();
 
+                    info!("{}", action.summarize());
```

Comment:
I'd move this down to line 324 ish, this means we've confirmed the `plan()` call returns steps that need to be actioned and the summary will then be relevant to the user.